enjoy.py

```diff
@@ -43,9 +43,13 @@ def main():
                         help='Normalize reward if applicable (trained with VecNormalize)')
     parser.add_argument('--seed', help='Random generator seed', type=int, default=0)
     parser.add_argument('--reward-log', help='Where to log reward', default='', type=str)
-
+    parser.add_argument('--gym-packages', type=str, nargs='+', help='Additional external Gym environemnt package modules to import (e.g. gym_minigrid)')
     args = parser.parse_args()
-
+    
+    for env_module in args.gym_packages:
+        import importlib
```

Comment:
Is there a reason why this import is not as the top ?

Reply:
Made the modification as much as local.   Moved to the top now.